addons/website/static/src/components/website_loader/website_loader.scss

```diff
@@ -33,4 +33,19 @@
             --TourPointer__width: 1em;
         }
     }
+
+    .o_website_loader_text-highlight {
+        color: lighten($o-enterprise-color, 20%);
+    }
```

Comment:
I lightened the color because the contrast (in terms of accessibility) was not sufficient with the base one.
Maybe I could directly use the dark mode color in there?

Reply:
Since its part of the specs I did not remove it, but I now apply the default primary color of the dark mode.
In order to avoid a clash of colors between the highlighted part of the title and the tour pointer, I also overrode its color with the dark mode one.